SPECS/kexec-tools/kexec-tools.spec

```diff
@@ -215,6 +218,11 @@ cp %{SOURCE109} $RPM_BUILD_ROOT/etc/kdump-adv-conf/kdump_dracut_modules/99earlyk
 chmod 755 $RPM_BUILD_ROOT/etc/kdump-adv-conf/kdump_dracut_modules/99earlykdump/%{remove_dracut_prefix %{SOURCE108}}
 chmod 755 $RPM_BUILD_ROOT/etc/kdump-adv-conf/kdump_dracut_modules/99earlykdump/%{remove_dracut_early_kdump_prefix %{SOURCE109}}
 
+# Add kexec-tools-specific boot configurations to /etc/default/grub.d
+# This configuration sets the crashkernel space allocated at boot
+# to the AzureLinux default value of 256M
+mkdir -p %{buildroot}%{_sysconfdir}/default/grub.d
+install -m 750 %{SOURCE30} %{buildroot}%{_sysconfdir}/default/grub.d/51_kexec_tools.cfg
```

Comment:
I probably should have commented on this in the grub PR also, but I don't think we should be setting 750 mode. At *most* we should allow an existing 027 umask to naturally affect the mode, but we definitely should not ever be explicitly setting 750. To be honest, we should stop using the 027 umask as well, but that's a totally separate discussion.